content/en/docs/tutorials/hello-minikube.md

```diff
@@ -38,7 +38,7 @@ This tutorial provides a container image built from the following files:
 
 {{< codenew language="conf" file="minikube/Dockerfile" >}}
 
-For more information, read the Docker documentation for [docker build](https://docs.docker.com/engine/reference/commandline/build/)
+For more information on the `docker build` command, read the [Docker documentation](https://docs.docker.com/engine/reference/commandline/build/).
```

Comment:
Question: Do we need this sentence? We don't seem to run the docker build command in the tutorial.

Reply:
I could be sold either way, since it's just a link I think it could be helpful if someone wanted to look more into docker build.